test/datasets_utils.py

```diff
@@ -129,7 +129,7 @@ def test_foo(self, config):
 
     @functools.wraps(test)
     def wrapper(self):
-        for config in self.CONFIGS or (self._DEFAULT_CONFIG,):
+        for config in self.CONFIGS or (self.DEFAULT_CONFIG,):
```

Comment:
This will not test on the `DEFAULT_CONFIG`, unless `CONFIGS` is left to None which is unlikely

Reply:
There are a few datasets without configurations that need to be tested:

- https://github.com/pytorch/vision/blob/c00c4d36d07552dbe0cc4fd399a9bf911bf99fcb/torchvision/datasets/coco.py#L22-L29
- https://github.com/pytorch/vision/blob/c00c4d36d07552dbe0cc4fd399a9bf911bf99fcb/torchvision/datasets/kinetics.py#L53-L56
- https://github.com/pytorch/vision/blob/c00c4d36d07552dbe0cc4fd399a9bf911bf99fcb/torchvision/datasets/sbu.py#L27-L33
- https://github.com/pytorch/vision/blob/c00c4d36d07552dbe0cc4fd399a9bf911bf99fcb/torchvision/datasets/semeion.py#L29-L35
- https://github.com/pytorch/vision/blob/c00c4d36d07552dbe0cc4fd399a9bf911bf99fcb/torchvision/datasets/flickr.py#L66-L72

In these cases the `@test_all_configs` decorator only tests the default configuration. Think of it as a crude reimplementation of `pytest.mark.parametrize("config", configs)`.

Reply:
Would it make sense to do

```suggestion
        for config in list(self.CONFIGS) + [self.DEFAULT_CONFIG]:
```

instead? I.e. always yield `DEFAULT_CONFIG`, no matter what. I think this would be consistent with the docstring for `CONFIG`: "Additional configs that should be tested" https://github.com/pytorch/vision/pull/3578/files#r595846984

(the suggestion might need to be updated considering hat `self.CONFIGS` can be None so it won't work as is)

Reply:
Yeah that seems like a good idea. Thanks for the suggestion!